x-pack/plugins/fleet/server/services/epm/archive/validation.ts

```diff
@@ -278,19 +279,23 @@ function parseAndVerifyVars(manifestVars: any[], location: string): RegistryVars
   }
   return vars;
 }
-function parseAndVerifyPolicyTemplates(manifest: any): RegistryPolicyTemplate[] {
+export function parseAndVerifyPolicyTemplates(
+  manifest: PackageSpecManifest
+): RegistryPolicyTemplate[] {
   const policyTemplates: RegistryPolicyTemplate[] = [];
   const manifestPolicyTemplates = manifest.policy_templates;
-  if (manifestPolicyTemplates && manifestPolicyTemplates > 0) {
+  if (manifestPolicyTemplates && manifestPolicyTemplates.length > 0) {
     manifestPolicyTemplates.forEach((policyTemplate: any) => {
       const { name, title: policyTemplateTitle, description, inputs, multiple } = policyTemplate;
-      if (!(name && policyTemplateTitle && description && inputs)) {
+      if (!(name && policyTemplateTitle && description)) {
```

Comment:
@skh I removed `input` being required in a policy template because it was causing all packages to fail during installation when getting the `endpoint` package from storage.  `endpoint` does not have any `inputs` in its `policy_templates`.  Is the package correct or is the spec incorrect?

<img width="1029" alt="Screen Shot 2020-12-10 at 7 32 09 PM" src="https://user-images.githubusercontent.com/1676003/101846177-6b49fa00-3b1e-11eb-865a-753550c8f3e2.png">

Reply:
I think you have it right. Let's follow spec as written and fix any bugs there

https://github.com/elastic/package-spec/blob/22bc835bcf0480c0973b2c6bbc86df1314e9c0ef/versions/1/manifest.spec.yml#L223

Reply:
In the parseAndVerify* methods I followed our existing types to decide which fields are mandatory. Spec and actual working package should take precedence though.